packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -288,7 +304,12 @@ class Auth {
      */
     async ready() {
         if (this.fetchedToken && this.fetchedToken !== '') {
-            this.pendingToken = Promise.resolve({...this.data, access_token: this.fetchedToken})
+            const {isGuest, customerId, usid} = this.decodeSlasJWT(this.fetchedToken)
+            this.set('access_token', this.fetchedToken)
+            this.set('customer_id', customerId)
+            this.set('usid', usid)
+            this.set('customer_type', isGuest ? 'guest' : 'registered')
+            this.pendingToken = Promise.resolve(this.data)
```

Comment:
What is the difference between `fetchedToken` and `pendingToken` and in what scenarios is each needed?